modules/serverless-kafka-source-kn.adoc

```diff
@@ -26,30 +27,35 @@ $ kn service create event-display \
 +
 [source,terminal]
 ----
-$ kn source kafka create mykafkasrc \
-    --servers my-cluster-kafka-bootstrap.kafka.svc:9092 \
-    --topics my-topic --consumergroup my-consumer-group \
+$ kn source kafka create <kafka_source_name> \
```

Comment:
One of the "Example outputs" in this file still references `source: /apis/v1/namespaces/default/kafkasources/mykafkasrc#my-topic` . Perhaps it should be something like `source: /apis/v1/namespaces/default/kafkasources/example-kafka-source#example-topic` ?

Reply:
Good call, thanks!